server/nodejs.go

```diff
@@ -95,10 +99,8 @@ var polyfilledBuiltInNodeModules = map[string]string{
 	"http":                "stream-http",
 	"https":               "https-browserify",
 	"os":                  "os-browserify/browser",
-	"path":                "path-browserify",
```

Comment:
do not change polyfilledBuiltInNodeModules, thoese polyfills are needed by broswers

Reply:
@ije alright got u lemme fix